backend/contracts/Quiz.sol

```diff
@@ -3,16 +3,19 @@ pragma solidity ^0.8.0;
 
 import "@oasisprotocol/sapphire-contracts/contracts/Sapphire.sol";
 import {EIP155Signer} from "@oasisprotocol/sapphire-contracts/contracts/EIP155Signer.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+// import "@openzeppelin/contracts/access/Ownable.sol";
 
-contract Quiz {
+
+contract Quiz is ERC721 {
     string constant errInvalidCoupon = "Invalid coupon";
     string constant errCouponExists = "Coupon already exists";
     string constant errWrongAnswer = "Wrong answer";
     string constant errWrongNumberOfAnswers = "Wrong number of answers";
     string constant errForbidden = "Access forbidden by contract policy";
     string constant errPayoutFailed = "Payout failed";
 
-    uint256 public constant COUPON_VALID =  type(uint256).max-1;
+    uint256 public constant COUPON_VALID = type(uint256).max-1;
```

Comment:
I see you're using a formatter. Can you add it and the linter to the package.json?

See for example https://github.com/oasisprotocol/sapphire-paratime/blob/main/contracts/package.json#L13-L20.

And then require the linter to pass in the github actions: https://github.com/oasisprotocol/sapphire-paratime/blob/main/.github/workflows/ci-lint.yaml#L81